src/cmdstan-guide/external_code.Rmd

```diff
@@ -4,19 +4,24 @@ The `--allow-undefined` flag can be passed to the call to stanc,
 which will allow undefined functions in the Stan language to be parsed
 without an error. We can then include a definition of the function in
 a C++ header file.
+
 This requires specifying two makefile variables:
+
 - `STANCFLAGS=--allow-undefined`
 - `USER_HEADER=<header_file.hpp>`, where `<header_file.hpp>` is the name of a header file that
-defines a function with the same name and signature in a namespace
-that is formed by concatenating the `class_name` argument to
-stanc documented above to the string `_namespace`
+  defines a function with the same name and signature. This function can appear in
```

Comment:
```suggestion
  defines a function with the same name and compatible signature. This function can appear in
```

Since the templating would allow for multiple signatures

Reply:
Good catch